pkg/controllers/disruption/multinodeconsolidation.go

```diff
@@ -38,13 +38,19 @@ const MultiNodeConsolidationType = "multi"
 
 type MultiNodeConsolidation struct {
 	consolidation
+	Validator
 }
 
 func NewMultiNodeConsolidation(consolidation consolidation) *MultiNodeConsolidation {
-	return &MultiNodeConsolidation{consolidation: consolidation}
+	return &MultiNodeConsolidation{
+		consolidation: consolidation,
+		Validator:     NewConsolidationValidator(consolidation),
+	}
 }
 
 func (m *MultiNodeConsolidation) ComputeCommand(ctx context.Context, disruptionBudgetMapping map[string]int, candidates ...*Candidate) (Command, scheduling.Results, error) {
+	// needed to reset the validator sync after every run
```

Comment:
This is a weird semantic -- why are we holding state in the validator like this?